packages/curve-common/src/widgets/Header/MobileHeader.tsx

```diff
@@ -52,7 +52,7 @@ export const MobileHeader = <TChainId extends number>({
   }, [startWalletConnection, closeSidebar])
 
   return (
-    <AppBar color="transparent" position="relative" sx={{ width: '100vw' }} ref={mainNavRef}>
+    <AppBar color="transparent" position="relative" ref={mainNavRef}>
```

Comment:
This was actually a bug, because it would go behind the scrollbar, causing a horizontal scroll. Caught by the new cypress tests.